docs/getting-started/project-setup.md

```diff
@@ -4,54 +4,40 @@ sidebar_label: Introduction
 description: Set yourself up for success
 ---
 
-## Rust
+## Overview
 
-First, you'll need Rust. To install Rust and the `cargo` build tool, follow the [official instructions](https://www.rust-lang.org/tools/install).
+Your local development environment will need several tools to compile, build, package and debug your Yew application.
 
-You also need to install the `wasm32-unknown-unknown` target to compile Rust to Wasm.
-If you're using rustup, you just need to run `rustup target add wasm32-unknown-unknown`.
+
+## Install Rust, Rustup and Cargo
+
+To install Rust compiler, Rustup package manager and the Cargo build tool, follow the [official instructions](https://www.rust-lang.org/tools/install).
 
 :::important
 The minimum supported Rust version (MSRV) for Yew is `1.45.0`. Older versions can cause unexpected issues accompanied by incomprehensible error messages.
 You can check your toolchain version using `rustup show` (under "active toolchain") or alternatively `rustc --version`. To update your toolchain, run `rustup update`.
 :::
 
-## **Wasm Build Tools**
+## Install WebAssembly target
+
+Rust compiles source code for a specific target.  For WebAssembly, this target is called "wasm32-unknown-unknown".  The following instruction will add this target to your development environment.
 
-Extra tooling is needed to facilitate the interop between WebAssembly and JavaScript. Additionally,
-depending on the tool you choose, they can help make deployment and packaging much less of a
-headache by generating all of the JavaScript code necessary to load and run your app's `.wasm`
-binary in a browser.
+`rustup target add wasm32-unknown-unknown`
 
-### [**`trunk`**](https://github.com/thedodd/trunk/)
+## Install Trunk
 
-A tool practically made for building Yew apps.
-It can build any `wasm-bindgen` based app and its design is inspired by rollup.js.
-With Trunk you don't need to have Node.js installed or touch any JavaScript code for that matter.
-It can bundle assets for your app and even ships with a Sass compiler.
+Trunk is the recommended tool for managing deployment and packaging, and will be used throughout the documentation and examples.
+See [Wasm Build Tools](./../more/wasm-build-tools.md) for more information on packaging and alternatives.
 
-All of our examples are built with Trunk.
+`cargo install trunk`
```

Comment:
```suggestion
```
# note that this might take a while to install, because it compiles everything from scratch
# Trunk also provides prebuilt binaries for a number of major package managers
# See https://trunkrs.dev/#install for further details
cargo install trunk
```
```